samigo/samigo-app/src/webapp/jsf/delivery/beginTakingAssessment.jsp

```diff
@@ -55,12 +55,19 @@
 <!-- DONE BUTTON FOR PREVIEW -->
 <h:panelGroup rendered="#{delivery.actionString=='previewAssessment'}">
   <div class="sak-banner-info">
-     <h:outputText value="#{deliveryMessages.ass_preview}" escape="false" />
+     <h:outputText value="#{deliveryMessages.ass_preview}" escape="true" />
+     <br>
+     <h:outputText value="#{deliveryMessages.seb_preview_info}" escape="false"/>
```

Comment:
Lets do this in a different way, it is fine to combine the message in the same key.

Reply:
I think it's more visible in a separate line, but it should only be rendered for `delivery.setSetup`